projects/demo/src/modules/components/slider/examples/import/declare-form.txt

```diff
@@ -1,12 +1,10 @@
-import {FormControl, FormGroup} from '@angular/forms';
+import {FormControl} from '@angular/forms';
```

Comment:
what do you think about using markdown? as it is in the example with the avatar demo